packages/accordion/src/AccordionContent.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+
+import { Wrapper } from './styled'
+export interface AccordionContentProps {
+  isExpanded: boolean
+  children: React.ReactNode
+  className?: string
+}
+
+const AccordionContent = React.memo((props: AccordionContentProps) => {
+  const self = useController()
+
+  return (
+    <Wrapper
```

Comment:
Would be good to add `aria-expanded` attribute to this panel for accessibility.

https://www.w3.org/WAI/ARIA/apg/patterns/accordion/